src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -130,21 +130,64 @@ module Types = struct
                 "The transaction has either been snarked, reached finality \
                  through consensus or has been dropped" ]
 
+  let consensus_time =
```

Comment:
I'm not opposed to this being here since it makes it easier to review graphql changes for the whole schema to be in one place, but curious why did it need to move as part of this PR? afaict there are still other parts of the schema in the Consensus module.

Reply:
There was a dependency cycle between Coda_lib and Consensus. Since some of the constants are not compile-time anymore, we'd have to fetch them from the context (for `startTime` and `endTime`). The others did not need the context. Will open an issue to move them to this module